### PR TITLE
chore: format verbiage on status page

### PR DESCRIPTION
**Pull request by PhearZero**
# Overview

- styles information panel in status page

# Issues

- https://algorandfoundation.atlassian.net/browse/ENG-499

### CODE REVIEW

File: src/xgov-dapp/src/features/status/InformationBox.tsx [hunk 1]
```diff
@@ -7,19 +7,28 @@ function InformationBox() {
...
         <Typography>
           During the Alpha phase governors were required to vote to maintain eligibility. If you missed a vote your eligibility was revoked.
-          For each Voting Session, the Voting Status confirms eligibility, where: You voted = Eligible You didn't vote = Not eligible At the
-          end of the Term Pool, the Algos forfeited by the ineligible xGovs are distributed amongst all eligible xGovs.
+          For each Voting Session, the Voting Status confirms eligibility, where:
+        </Typography>
+        <Typography>You voted = Eligible</Typography>
+        <Typography>You didn't vote = Not eligible</Typography>
+        <Typography className="my-3">
+          At the end of the Term Pool, the Algos forfeited by the ineligible xGovs are distributed amongst all eligible xGovs.
```
Comment (kylebeee) [inline src/xgov-dapp/src/features/status/InformationBox.tsx:18]: Algo instead of Algos

**Review by kylebeee — APPROVED** [review]
LGTM other than remove the S

File: src/xgov-dapp/src/features/status/InformationBox.tsx [hunk 2]
```diff
@@ -7,19 +7,28 @@ function InformationBox() {
...
-
         <Typography className="my-3">
           <strong>Term Pools</strong>
         </Typography>
-        Term pools (TP) funds are held by the Algorand Foundation for 12 months from inception. Once a Term Pool matures the funds are
-        transferred to the original wallet used in governance or, if indicated at the time of commitment, to the governor's controller
-        address. The Duration column displays the creation and maturation dates for each term pool. Once a term pool matures, funds are
-        distributed within 14 days.
+        <Typography>
+          Term pools (TP) funds are held by the Algorand Foundation for 12 months from inception. Once a Term Pool matures the funds are
```
Comment (tasosbit) [inline src/xgov-dapp/src/features/status/InformationBox.tsx:24]: Should this be `Term pool (TP) funds` (not plural?)

**Comment by tasosbit** [comment]
There are also a few unused import warnings if you could get to them

File: src/xgov-dapp/src/features/status/InformationBox.tsx (hunk 2 above)
Comment (PhearZero) [reply src/xgov-dapp/src/features/status/InformationBox.tsx:24]: Maybe possessive? `Term pool's`. I can ask Adri

**Comment by PhearZero** [comment]
> There are also a few unused import warnings if you could get to them, e.g. `Skeleton`

@tasosbit The project is winding down, there are quite a bit of chores/todos that we won't be able to get to. This is the last change for the system as it is sunset (other than updating terms status as they are distributed)

**Pull request merged**